addons/website_event_meet_quiz/views/event_meet_templates.xml

```diff
@@ -25,7 +25,7 @@
                     t-att-src="image_data_uri(visitor['visitor'].partner_image) if visitor['visitor'].partner_image else '/web/static/img/user_placeholder.jpg'"
                     t-att-alt="visitor['visitor'].display_name"/>
                 <div class="o_we_leaderboard_name">
-                    <span t-if="visitor['visitor'] == current_visitor and not visitor['visitor'].name" class="font-weight-bold">You</span>
+                    <span t-if="visitor['visitor'] == current_visitor and not visitor['visitor'].partner_id" class="font-weight-bold">You</span>
```

Comment:
This change is probably wrong. If there is a partner and this is you, we want to display "You".